CHANGES.md

```diff
@@ -998,15 +998,18 @@ OpenSSL 3.0
    MD5_Final, MD5_Transform, MDC2, MDC2_Init, MDC2_Update, MDC2_Final,
    RIPEMD160, RIPEMD160_Init, RIPEMD160_Update, RIPEMD160_Final,
    RIPEMD160_Transform, SHA1_Init, SHA1_Update, SHA1_Final, SHA1_Transform,
-   SHA224_Init, SHA224_Update, SHA224_Final, SHA224_Transform, SHA256_Init,
-   SHA256_Update, SHA256_Final, SHA256_Transform, SHA384, SHA384_Init,
-   SHA384_Update, SHA384_Final, SHA512, SHA512_Init, SHA512_Update,
-   SHA512_Final, SHA512_Transform, WHIRLPOOL, WHIRLPOOL_Init,
+   SHA224_Init, SHA224_Update, SHA224_Final, SHA224_Transform,
+   SHA256_Init, SHA256_Update, SHA256_Final, SHA256_Transform,
+   SHA384_Init, SHA384_Update, SHA384_Final,
+   SHA512_Init, SHA512_Update, SHA512_Final, SHA512_Transform,
+   WHIRLPOOL, WHIRLPOOL_Init,
    WHIRLPOOL_Update, WHIRLPOOL_BitUpdate and WHIRLPOOL_Final.
 
    Use of these low level functions has been informally discouraged
    for a long time.  Applications should use the EVP_DigestInit_ex(3),
-   EVP_DigestUpdate(3) and EVP_DigestFinal_ex(3) functions instead.
+   EVP_DigestUpdate(3) and EVP_DigestFinal_ex(3) functions instead or simply,
```

Comment:
This doesn't read correctly.. Maybe something like

init(), update() and final() functions instead. Alternatively EVP_Q_digest() can be used().
SHA1,SHA224 .... have changed from functions to macros that use EVP_Q_digest().

Reply:
Good point. I've re-worked the 2nd half of the paragraph to:
```
   Alternatively, the quick one-shot function L<EVP_Q_digest(3)> can be used.
   SHA1, SHA224, SHA256, SHA384 and SHA512 have changed from functions to macros
   like this: (EVP_Q_digest(d, n, md, NULL, NULL, "SHA256", NULL) ? md : NULL).
```
which reflects the situation both more succinctly and precisely.